packages/block-library/package.json

```diff
@@ -46,6 +46,7 @@
 		"@wordpress/date": "file:../date",
 		"@wordpress/deprecated": "file:../deprecated",
 		"@wordpress/dom": "file:../dom",
+		"@wordpress/editor": "file:../editor",
```

Comment:
This is probably the major downside of this PR 😬  In the past, we were keen to avoid having the blocks depend on the editor.

As for the reason for this change, see this inline comment: https://github.com/WordPress/gutenberg/pull/40898/files#diff-e1efd67de68dbd5ce8acea06af13216e3c2455c7b87c16d31cea7df4f282a9fbR209-R211